AimBot/Process.h

```diff
@@ -0,0 +1,28 @@
+#pragma once
+
+class Process
+{
+	std::unique_ptr<HANDLE> process;
```

Comment:
No need to store HANDLE in unique_ptr. This is essentially just a number (pointer). 
`HANDLE process = INVALID_HANDLE_VALUE`

and then

```
	~Process()
 	{
                if(process != INVALID_HANDLE_VALUE)
 		    CloseHandle(process);
 	}
```